src/snowcli/cli/snowpark/procedure.py

```diff
@@ -159,7 +159,6 @@ def procedure_update(
     replace: bool = typer.Option(
         False,
         "--replace-always",
-        "-r",
```

Comment:
Why are we removing it?

Reply:
It's a conflict with `-r` ued for return value